### ISSUE TITLE
Reduce code duplication in Xcode/TranslateTasks

### ISSUE THREAD

**Issue by advayDev1** [j2objc-contrib/j2objc-gradle#461]
http://nemo.sonarqube.org/drilldown/measures/880443?highlight=duplicated_lines_density&metric=duplicated_lines&rids%5B%5D=880450

**Comment by confile** [comment]
@advayDev1 What is this? Does it has to do with ProGuard?

**Comment by advayDev1** [comment]
No.  Check the link.  This is a code health improvement to the plugin and has no effect on the output code/library.

**Comment by advayDev1** [comment]
Low priority code quality issue IMO.  Bigger fish to fry.
